.vscode/Unified.code-workspace.example.jsonc

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Please make this a 2-space indented file.

Reply:
👍🏻and I added settings which will do this automatically if you use this workspace